test/public/tests.js

```diff
@@ -12,19 +12,21 @@ if (typeof module !== 'undefined' && module.exports) {
   imageSrc = function (filename) { return filename }
 }
 
-tests['clearRect()'] = function (ctx) {
+tests['clearRect()'] = function (ctx, done) {
```

Comment:
Instead of having every test accepting a `done` parameter you could simply test the arity with `const hasCallback = tests['clearRect()'].length === 2`, that way you don't have to modify all the existing tests, and that is in line with how mocha does it.